share/lib/python/scripts/_binwrapper.py

```diff
@@ -63,22 +63,20 @@ def _check_cpp_compiler_version():
 
 def _config_exe(exe_name):
     """Sets the environment to run the real executable (returned)"""
+    NRN_PREFIX = None
+    package_names = ["neuron", "neuron-nightly"]
 
-    package_name = "neuron"
+    # Search for NEURON package installation location using importlib.metadata
+    for dist in distributions():
```

Comment:
I think it's a bit clearer to use `metadata("neuron-nightly")` in a `try..except PackageNotFoundError` here as we did [for NMODL](https://github.com/BlueBrain/nmodl/commit/cd4edec63bc85a69b57a1bce74daa27cad28052a#diff-8709632f13f1a9d1ad5eed3d6a66cd3d5d80f29f42c1eddb745c222e030762c3).

Reply:
Hi @JCGoran, ah great so you already did this change for NMODL.

Reply:
I actually didn't understand this part. I don't know how the code should behave for neuron-nightly. Do you mind adding this change here?